Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Card.css';
+
+
+const CardBanner = () =>{
+    return (
+        <div className='bannerIMGContainer'></div>
```

Comment:
Is your image working on your end? You may need to find a different way to accomplish this.